src/client/Microsoft.Identity.Client/Http/HttpManager.cs

```diff
@@ -145,19 +145,17 @@ private HttpRequestMessage CreateRequestMessage(Uri endpoint, IDictionary<string
             {
                 if (cancellationToken.IsCancellationRequested)
                 {
-                    logger.Info("The HTTP request was cancelled. ");
+                    logger.Error("The HTTP request was cancelled. ");
```

Comment:
Not an error, at most a warning. Very common to cancel the request in AcquireTokenForDeviceCode